src/views/catalog/wizard/tabs/scripts/components/Sysprep.tsx

```diff
@@ -1,4 +1,5 @@
-import * as React from 'react';
+import React, { FC } from 'react';
+import { getSysprepConfigMapName } from 'src/views/templates/details/tabs/scripts/components/SysPrepItem/sysprep-utils';
```

Comment:
can be moved to a more shared location out of templates

Reply:
Definitely!

Reply:
Updates here: https://github.com/kubevirt-ui/kubevirt-plugin/pull/1913